contracts/interfaces/IEntryPoint.sol

```diff
@@ -9,12 +9,15 @@ pragma solidity ^0.8.12;
 /* solhint-disable no-inline-assembly */
 /* solhint-disable reason-string */
 
+// we also require '@gnosis.pm/safe-contracts' and both libraries have 'IERC165.sol', leading to conflicts
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol" as OpenZeppelin;
+
 import "./UserOperation.sol";
 import "./IStakeManager.sol";
 import "./IAggregator.sol";
 import "./INonceManager.sol";
 
-interface IEntryPoint is IStakeManager, INonceManager {
+interface IEntryPoint is IStakeManager, INonceManager, OpenZeppelin.IERC165 {
```

Comment:
why should the IEntryPoint inherit from ERC165 ? (you later manually remove it, so maybe it should not be in the interface itself...

Reply:
actually, maybe its good, so a bundler can work only with IEntryPoint, not with EntryPoint contract itself.